src/main/java/at/hannibal2/skyhanni/features/fishing/FishingTimer.kt

```diff
@@ -51,7 +51,7 @@ object FishingTimer {
 
     private var mobsToFind = 0
 
-    private val recentMobs = TimeLimitedSet<Mob>(2.seconds)
+    private var recentMobs = TimeLimitedSet<Mob>(2.seconds)
```

Comment:
Is there a reason to make those var?

Reply:
After my commit the `object += value` requires this for the add function. Not sure if this is the correct solution though

Reply:
reverted the changes in that commit that hanni made